imports/startup/client/useraccounts-configuration.js

```diff
@@ -1,15 +1,16 @@
 import { AccountsTemplates } from 'meteor/useraccounts:core';
+import { TAPi18n } from 'meteor/tap:i18n'
 
 AccountsTemplates.configure({
   showForgotPasswordLink: true,
   texts: {
     errors: {
-      loginForbidden: 'Incorrect username or password',
-      pwdMismatch: 'Passwords don\'t match',
+      loginForbidden: TAPi18n.__('Incorrect username or password'),
```

Comment:
I mean I think that'll probably set the internal strings (we don't actually have any way to change locale right now though). But the strings we pass in will need to be translated.